app/views/layouts/application.html.haml

```diff
@@ -3,7 +3,13 @@
   %head
     %meta{charset: 'utf-8'}
     %meta{content: 'width=device-width, initial-scale=1.0', name: 'viewport'}
+
+    %meta{name: 'title', content: 'hackEDU'}
+    %meta{name: 'description', content: 'The international movement of free student-led coding clubs.'}
+    %meta{name: 'og:image', content: "#{ image_path('logo.png') }"}
+
     %title= full_title(yield(:title))
+
```

Comment:
Just created #166 to track this.

On Wed, Sep 9, 2015 at 5:56 PM, Max Wofford <notifications@github.com>
wrote:

> In app/views/layouts/application.html.haml
> <https://github.com/hackedu/website/pull/157#discussion_r39115056>:
>
> >      %title= full_title(yield(:title))
> > +
>
> Okay. Should we create an issue for it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/website/pull/157/files#r39115056>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us